docs/dev/nightly_data_builds.rst

```diff
@@ -121,64 +121,56 @@ permissions.
 How to access the nightly build outputs from AWS
 ------------------------------------------------
 You can download the outputs from a successful nightly build data directly from the
-``s3://pudl.catalyst.coop`` bucket. To do this, you'll
-need to `follow the instructions
-<https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`__
-for installing the AWS CLI tool.
-
-To test that the AWS CLI tool and the bucket are working properly, run:
+``s3://pudl.catalyst.coop`` bucket using the ``gcloud storage`` CLI, which can
+access both GCS and S3 storage buckets.
```

Comment:
Is there a reason we were having people use the AWS CLI rather than `gcloud` (which can handle both GCS and S3 and which is part of our environment and doesn't need to be installed separately)?  It seemed like maybe a holdover so I updated it all to use the google tooling.